### DISCUSSION TITLE
Use pub_updator for serverpod_cli

### DISCUSSION THREAD

**Discussion by shan-shaji** [serverpod/serverpod#606] [Ideas]
https://pub.dev/packages/pub_updater

I believe it would be nice to give the developers a prompt to update the package if a new version of the package is updated in pub.dev.

**Comment by vlidholt** [comment]
That's a cool idea. I added an issue for it. :)

**Reply by shan-shaji** [reply]
Shall i work on it?

**Reply by shan-shaji** [reply]
@vlidholt

**Reply by vlidholt** [reply]
Feel free to do a PR :)
